src/interface/mopac_version.F90

```diff
@@ -16,7 +16,9 @@
 
 subroutine mopac_version(version)
 !dec$ attributes dllexport :: mopac_version
-  character :: version*12
-  version = "22.0.0-"
+  character :: version*20
+#ifdef MOPAC_VERSION_FULL
+  version = MOPAC_VERSION_FULL
+#endif
```

Comment:
`version` is not initialized in the other case?

Reply:
Should it just be initialized to `unknown` like `git_hash`, `os`, etc

Reply:
Yes, this was an oversight. The internal variable for the version is set to `unknown`, but the `mop_version` subroutine isn't sync'ed with that variable. I'm about to revise MOPAC's very minimal interface, and I will fix this.